main.go

```diff
@@ -50,7 +42,22 @@ func getProtectedResource(c *gin.Context) {
 
 func main() {
 	router := gin.Default()
-	router.GET("/protected", getProtectedResource)
+	svc := &Service{}
+
+	err := godotenv.Load(".env")
+	if err != nil {
+		fmt.Println("Failed to load .env file")
+	}
+	lndClient, err := lnd.NewLNDclient(lnd.LNDoptions{
```

Comment:
In this place you can log a `Fatal` error. If the LND connection does not work, the app should crash here.